doc/other/support.rst

```diff
@@ -4,4 +4,4 @@ Support
 Need help? Create a Github issue or ping the Phpactor twitter account:
 
 -  `Github Issues <https://github.com/phpactor/phpactor/issues>`__
--  Twitter [@phpactor](https://twitter.com/phpactor)
+-  `Twitter [@phpactor] <https://twitter.com/phpactor>`__
```

Comment:
I'm posting more on https://phpc.social/@phpactor than Twitter currently, probably worth adding that now or later.